numpy/core/src/multiarray/conversion_utils.c

```diff
@@ -125,27 +196,34 @@ PyArray_IntpConverter(PyObject *obj, PyArray_Dims *seq)
     if (len < 0) {
         PyErr_SetString(PyExc_TypeError,
                 "expected sequence object with len >= 0 or a single integer");
+        Py_DECREF(seq_obj);
         return NPY_FAIL;
     }
     if (len > NPY_MAXDIMS) {
-        PyErr_Format(PyExc_ValueError, "maximum supported dimension for an ndarray is %d"
-                     ", found %d", NPY_MAXDIMS, len);
+        PyErr_Format(PyExc_ValueError, "maximum supported dimension for an "
+                    "ndarray is %d, found %d", NPY_MAXDIMS, len);
```

Comment:
NIT: I/we currently prefer formatting these as (the continuation is 8 space indented)
```
PyErr_Format(PyExc_ValueError,
        "maximum supported dimension for an ndarray is %d, found %d",
        NPY_MAXDIMS, len);
```

Reply:
Fixed in d549b0f, let me know if you spot something similar which I did not see.

Reply:
Oh, I was going for a line break after the `PyExc_ValueError`, keeps the string together.  To be fair, we only use that style fairly consistently for errors.